src/components/Annotation.tsx

```diff
@@ -1,17 +1,17 @@
 import React from 'react';
-import { suomifiTheme } from 'suomifi-ui-components';
+import { defaultTokens } from 'suomifi-ui-components';
```

Comment:
Now that I see this imported here, it kind of rings a bell... should this be named suomiFiTokens or suomifiDesignTokens rather than defaultTokens? Not a matter to be handled in this PR though. 😅

Reply:
https://github.com/vrk-kpa/suomifi-ui-components/issues/219